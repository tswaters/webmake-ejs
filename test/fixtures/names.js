'use strict';
module.exports = {
  'names': ['loki', 'tobi', 'jane']
};
