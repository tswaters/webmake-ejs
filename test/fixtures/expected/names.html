<ul>
  
    <li>loki</li>
  
    <li>tobi</li>
  
    <li>jane</li>
  
</ul>